packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -440,7 +440,7 @@ class FileCollection extends DocumentCollection {
   async copy(id, name, dirId) {
     const params = {
       name: name === undefined ? undefined : sanitizeAndValidateFileName(name),
-      dirId
+      DirID: dirId
```

Comment:
Does it mean this method never worked? Is it used somewhere?

Reply:
This was not used by Drive where there was duplicate function. I follow the [documentation](https://docs.cozy.io/en/cozy-stack/files/#post-filesfile-idcopy) after realising on Nextcloud that the stack was sensitive to the case in these arguments.